src/test/java/com/github/underscore/ChainingTest.java

```diff
@@ -415,6 +415,34 @@ public Integer apply(Integer accum, Integer length) {
     }
 
     @Test
+    @SuppressWarnings("unchecked")
+    public void chain8() {
+        final List<Comparable> result = $.chain($.chain($.class.getDeclaredMethods())
+            .reduce(new FunctionAccum<List<String>, Method>() {
```

Comment:
Hi,

Thanks for your improvements!

It may me map() method here instead of reduce().